src/protocols/bdx/TransferFacilitator.h

```diff
@@ -107,18 +90,13 @@ class Responder : public TransferFacilitator
 {
 public:
     /**
-     * Initialize the TransferSession state machine to be ready for an incoming transfer request, and start the polling timer.
-     *
-     * @param[in] layer           A System::Layer pointer to use to start the polling timer
+     * Initialize the TransferSession state machine to be ready for an incoming transfer request and set the callback to get the
+     * output event
```

Comment:
Is the "callback to get the output event" something the consumer of this API needs to know about?

Reply:
no this was a find replace of the polling timer comments. will remove

Reply:
done